README.md

```diff
@@ -25,60 +46,77 @@ auto_activate_base: false
 anaconda_upload: false
 ```
 
-If not loading Anaconda by default, due to
-[``conda/conda`` issue #7980](https://github.com/conda/conda/issues/7980),
-you will need to source something similar to:
-
-```
-#!/bin/bash
-
-PATH=~/Development/python/anaconda3/bin:$PATH
-. ~/Development/python/anaconda3/etc/profile.d/conda.sh
-```
-
-The work directory of a conda environment can be accessed via an environment variable, allowing us thus
-to build up the Cylc UI directory as follows: `${CONDA_PREFIX}/work/cylc-ui`.
-
-## Testing
-
-You need to enable the `kinow` channel for this test to work (see `.condarc` file above).
-
-Then create a new Conda environment, e.g. `cylc1`.
+Create a new Conda environment, e.g. `cylc1`.
 
 - `conda create -n cylc1`
 - `conda activate cylc1`
 
-Now you only need to install the placeholder package "cylc". You can refer to the `recipes/cylc/meta.yaml`
-for its contents. But this is basically installing conda packages for `metomi-isodatetime`, `cylc-flow`,
-`cylc-uiserver`, and unzipping `cylc-ui` into the conda environment work directory (`$PREFIX` conda build env
-var).
+Install Cylc
 
-- `conda install cylc` (you can wait for the confirmation question, where you have a chance to see all the packages being installed, including a few cylc* and metomi*)
+- `conda install cylc`
 
-At this point you can test your environment if you would like, with some commands like:
+You should now be able to run Cylc commands, e.g.:
 
 - `cylc --version`
-- `cylc run --no-detach five` (or any other suite you use for testing)
+- `cylc run --no-detach my.suite`
 - `cylc-uiserver --help`
-- `isodatetime`
-
-Finally, if you found nothing wrong with your environment, start the hub with:
 
-- `jupyterhub --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner" --Spawner.args="['-s', '${CONDA_PREFIX}/work/cylc-ui']" --Spawner.cmd="cylc-uiserver" --JupyterHub.logo_file="${CONDA_PREFIX}/work/cylc-ui/img/logo.png"`
-
-Voilà 🎉. Go to `http://localhost:8000`, log in with your local user credentials, and enjoy Cylc 8 alpha!
-
-You can now remove the environment if you would like.
+Start the hub (jupyterhub gets installed with the "cylc" package):
+
+- `jupyterhub
+    --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner"
+    --Spawner.args="['-s', '${CONDA_PREFIX}/work/cylc-ui']"
+    --Spawner.cmd="cylc-uiserver"
+    --JupyterHub.logo_file="${CONDA_PREFIX}/work/cylc-ui/img/logo.png"`
+
+Voilà 🎉. Go to `http://localhost:8000`, log in wto the Hub ith your local user
```

Comment:
```suggestion
Voilà 🎉. Go to `http://localhost:8000`, log in to the Hub with your local user
```

Reply:
Ha, that's a weird typo! I must have started typing in the middle of a word :grimacing: